synapse/lib/webapp.py

```diff
@@ -102,14 +102,14 @@ def getFooByBar(self, bar):
         wapp.main()
 
     '''
-    def __init__(self, **settings):
+    def __init__(self, app_config={}, srv_config={}):
```

Comment:
based on conversation - its probably best to make this forwards compatible by retaining **settings and doing the following:

```def __init__(self, app_config={}, srv_config={}, **settings):```

That way we can pull things out in the future (for example, to support looking at making webapp confiable ie. #330 )